### PR TITLE
Small tweaks to docs to make them more user friendly

### PR DESCRIPTION
**Pull request by zvecr**
## Description

Summary of the changes,
* Split out newbs testing_debugging to faq pages
  * Reason being generally new starters are not going to start with hid_listen type stuff
*  Strip down the wording of some setup sections
  * Mostly round the "download software" section so users can see the good stuff a little easier
* Call out various sections with a pop of colour to draw users to the key parts of the page

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
 
 ## Get Started
 
-Totally new to QMK? There are two ways to get started:
+<div class="flex-container">
+
+?> **Basic** [QMK Configurator](newbs_building_firmware_configurator.md) <br>
+User friendly graphical interfaces, no programming knowledge required.
+
+?> **Advanced** [Use The Source](newbs.md) <br>
```
Comment (drashna) [inline docs/README.md:14]: Mostly, just a joke, but couldn't pass up. 
```suggestion
?> **Advanced** [Use The Source, Luke](newbs.md) <br> 
```

File: docs/faq_debug.md [hunk 2]
```diff
@@ -2,147 +2,118 @@
...
 
+When porting, or when attempting to diagnose pcb issues, it can be useful to know if a keypress is scanned correctly. To enable logging for this scenario, add the following code to your keymaps `keymap.c`
 
-## TrackPoint Needs Reset Circuit (PS/2 Mouse Support)
-Without reset circuit you will have inconsistent result due to improper initialize of the hardware. See circuit schematic of TPM754.
+```c
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  // If console is enabled, it will print the matrix position and status of each key pressed
+#ifdef CONSOLE_ENABLE
+    uprintf("KL: kc: %u, col: %u, row: %u, pressed: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed);
```
Comment (drashna) [inline docs/faq_debug.md:58]: This is a bit more complicated... but it formats the keycode as a hex (which is more useful when comparing to code), and adds additional useful structures from the keyrecord.
```suggestion
    uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %b, time: %u, interrupt: %b, count: %u\n\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
```

Comment (zvecr) [reply docs/faq_debug.md:58]: Works on vusb/lufa/chibios?

Comment (drashna) [reply docs/faq_debug.md:58]: It works on LUFA/ChibiOS, but I can't verify on VUSB, as I don't own any boards/controllers for VUSB.

Comment (fauxpark) [reply docs/faq_debug.md:58]: The printf code is shared between LUFA and VUSB boards, so it should work.

**Review by drashna — APPROVED** [review]

**Pull request merged**